tests/python/gaia-ui-tests/gaiatest/tests/graphics/RTL/test_Contacts_RTL.py

```diff
@@ -0,0 +1,64 @@
+from gaiatest.gaia_graphics_test import GaiaImageCompareTestCase
+from gaiatest.apps.contacts.app import Contacts
+
+class TestContactsRTL(GaiaImageCompareTestCase):
+
+    def setUp(self):
+       GaiaImageCompareTestCase.setUp(self)
+
+       self.push_resource('IMG_0001.jpg')
+
+    def test_contacts_app(self):
+
+        contacts = Contacts(self.marionette)
+        contacts.launch()
+        self.take_screenshot('contacts')
+        view_settings = contacts.tap_settings()
+        self.take_screenshot('contacts-setting')
+        view_settings.tap_import_contacts()
+        self.take_screenshot('contacts-import')
+        view_settings.tap_import_from_sim()
+        self.take_screenshot('contacts-sim_import')
+        view_settings.tap_back_from_import_contacts()
+        self.take_screenshot('contacts-back_to_settings')
+        view_settings.tap_order_by_last_name()
+        self.take_screenshot('contacts-orderby_lastname')
+        view_settings.tap_done()
+        self.take_screenshot('contacts-tap_done')
+        for i in range(0, 2):
+            GaiaImageCompareTestCase.scroll(self.marionette, 'down', contacts.screen_element.size['height'], screen=contacts.screen_element)
+            self.take_screenshot('main')
+
+        some_contacts = contacts.tap_new_contact()
+        self.take_screenshot('contacts-new_contact')
+        some_contacts.type_given_name("testing")
+        self.take_screenshot('contacts-given_name')
+        some_contacts.type_family_name("tester")
+        self.take_screenshot('contacts-famillname')
+        some_contacts.type_phone("52306148")
+        self.take_screenshot('contacts-phone')
+        some_contacts.type_email("test-test@gmail.com")
+        self.take_screenshot('contacts-phone')
+        some_contacts.type_street("Rue tester")
+        self.take_screenshot('contacts-Street')
+        some_contacts.type_city('Manouba')
+        self.take_screenshot('contacts-city')
+        some_contacts.type_country('TestCountry')
+        self.take_screenshot('contacts-country')
+        some_contacts.type_zip_code('2563')
+        self.take_screenshot('contacts-zip_code')
+        some_contacts.type_comment('this comment for test')
+        self.take_screenshot('contacts-comment')
+        activities_list = some_contacts.tap_picture()
+        self.take_screenshot('contacts-picture')
+        gallery = activities_list.tap_gallery()
+        self.take_screenshot('contacts-from_gallery')
+        gallery.wait_for_thumbnails_to_load()
```

Comment:
oh actually, we don't need to take screenshots after entering each line- that would be too much.  since we are only checking for RTL work of the app, we don't need to to take screenshots of our input.  (I'm guessing you are entering a new contact so we can see the contact info view?)